BUILD

```diff
@@ -1,35 +1,20 @@
 # gRPC Bazel BUILD file.
 #
-# Copyright 2016, Google Inc.
-# All rights reserved.
+# Copyright 2016 gRPC authors.
```

Comment:
I've based this on https://github.com/grpc/grpc/issues/11295 and 
https://www.apache.org/licenses/LICENSE-2.0   (java also mentions 'All Rights Reserved' which seems unnecessary, so I decided to follow the official guidelines).
If needed, we can update in the future, but I'll merge for now, because updating all the licenses is suprisingly hard to get right given the number of files & languages in our repo and how quickly they change.